### PR TITLE
37 create global styling

### LINKED ISSUE #33: Update layout of LiveviewInput component
Good luck, there are many buttons but make it look nice :)
Maybe add an indicator to show which state autobrewing is in other than the buttons just being disabled.

### LINKED ISSUE #37: Create global styling

### PR DESCRIPTION
**Pull request by Kaszz**
Closes #37 #33
Changed the design of EVERYTHING. 
It is now responsive and looks good on devices as small as IPhone 6/7/8.
The files have also been structured in folders.

Gif showing the different pages a little bit.

Gif showing responsiveness.

We recommend that you pull it and play around.

### CODE REVIEW

**Review by Teyson — APPROVED** [review]

File: src/components/views/batchview/Batches.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            setSelectedBatchID(searchVal);
+            fetchChosenBatch(searchVal);
+        } else {
+            alert("Please provide a valid ID");
+        }
+    };
+    
+    const setGraphData = (id) => {
+        if (id === 'State') {
+            console.log("State chosen");
```
Comment (LeaChr96) [inline src/components/views/batchview/Batches.js]:
```suggestion

```

File: src/components/views/liveview/Liveview.js [hunk 2]
```diff
@@ -1,101 +1,139 @@
...
+			humidity: 0,
+			normSpeed: 0,
+			maintenance: 0,
+			state: "STOPPED",
+			ingredients: {
+				barley: 35000,
+				hops: 35000,
+				malt: 35000,
+				wheat: 35000,
+				yeast: 35000
```
Comment (LeaChr96) [inline src/components/views/liveview/Liveview.js:29]: I know this is unrelated. 
Ideally, we would probably not have this hardcoded here. But it's probably not worth changing now.

**Review by LeaChr96 — APPROVED** [review]
The code readability suffers a bit from having been refactored. I'd recommend using the auto formatter in VS Code or similarly.

**Comment by LeaChr96** [comment]
image

If the back-end is not turned on the UI breaks a bit on the schedule page.

Not sure that it's related to this pull request or that we even care.

**Comment by LeaChr96** [comment]
image
The text is slightly off in some cases at the size iPhone X. The search text could probably be made slightly smaller. Making the list fit is probably going to be more problematic.

**Comment by Kaszz** [comment]
> image
> 
> If the back-end is not turned on the UI breaks a bit on the schedule page.
> 
> Not sure that it's related to this pull request or that we even care.

There was a purpose with this in that the element is only displayed when you actually have some available products to actually make a list. Don't remember the exact problem or reasoning but this was done according to solve another bigger problem :/

**Comment by Kechr** [comment]
> The code readability suffers a bit from having been refactored. I'd recommend using the auto formatter in VS Code or similarly.

Any specific files you are referring to? It looks 💩 on git, but okay in vs code for me.

**Review by Wylardt — APPROVED** [review]

**Comment by LeaChr96** [comment]
> > The code readability suffers a bit from having been refactored. I'd recommend using the auto formatter in VS Code or similarly.
> 
> Any specific files you are referring to? It looks 💩 on git, but okay in vs code for me.

It's nothing much. Just some spacing in some classes, but it's rather minor.

**Review by dottore1 — APPROVED** [review]

**Pull request merged**